geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommandDUnitTest.java

```diff
@@ -15,290 +15,150 @@
-    String str2 = "Key         : k1";
-    String str3 = "Key Class   : java.lang.String";
-    String str4 = "Value Class : java.lang.String";
-    String str5 = "Old Value   : <NULL>";
-
-    assertTrue(
-        commandResultToString(cmdResult).contains("Region \"/" + region46391 + "\" created on"));
-
-    assertTrue(commandResultToString(cmdResult2).contains(str1));
-    assertTrue(commandResultToString(cmdResult2).contains(str2));
-    assertTrue(commandResultToString(cmdResult2).contains(str3));
-    assertTrue(commandResultToString(cmdResult2).contains(str4));
-    assertTrue(commandResultToString(cmdResult2).contains(str5));
+  @Test
+  public void testCreateRegionWithNoCompressor() throws Exception {
+    String regionName = testName.getMethodName();
+    gfsh.executeAndVerifyCommand("create region --name=" + regionName + " --type=REPLICATE");
+
+    server.invoke(() -> {
+      Cache cache = LocatorServerStartupRule.serverStarter.getCache();
+      Region region = cache.getRegion(regionName);
+      assertThat(region).isNotNull();
+      assertThat(region.getAttributes().getCompressor()).isNull();
+    });
   }
 
-  /**
-   * Test Description 1. Deploy a JAR with Custom Partition Resolver 2. Create Region with Partition
-   * Resolver 3. Region should get created with no Errors 4. Verify Region Partition Attributes for
-   * Partition Resolver
-   */
   @Test
-  public void testCreateRegionWithPartitionResolver() throws IOException {
-    setUpJmxManagerOnVm0ThenConnect(null);
-    VM vm = Host.getHost(0).getVM(1);
-    // Create a cache in vm 1
-    vm.invoke(() -> assertNotNull(getCache()));
-
+  public void testCreateRegionWithPartitionResolver() throws Exception {
+    String regionName = testName.getMethodName();
     ClassBuilder classBuilder = new ClassBuilder();
     // classBuilder.addToClassPath(".");
-    final File prJarFile = new File(temporaryFolder.getRoot().getCanonicalPath() + File.separator,
-        "myPartitionResolver.jar");
-    this.filesToBeDeleted.add(prJarFile.getAbsolutePath());
-    String PR_STRING = " package com.cadrdunit;"
-        + " public class TestPartitionResolver implements org.apache.geode.cache.PartitionResolver { "
+    final File prJarFile = tmpDir.newFile("myPartitionResolver.jar");
+    String PR_STRING = "package io.pivotal; "
+        + "public class TestPartitionResolver implements org.apache.geode.cache.PartitionResolver { "
         + "   @Override" + "   public void close() {" + "   }" + "   @Override"
         + "   public Object getRoutingObject(org.apache.geode.cache.EntryOperation opDetails) { "
         + "    return null; " + "   }" + "   @Override" + "   public String getName() { "
         + "    return \"TestPartitionResolver\";" + "   }" + " }";
     byte[] jarBytes =
-        classBuilder.createJarFromClassContent("com/cadrdunit/TestPartitionResolver", PR_STRING);
+        classBuilder.createJarFromClassContent("io/pivotal/TestPartitionResolver", PR_STRING);
```

Comment:
We have a new utility class to replace classBuilder that will make this a bit simpler: 

```
  String PR_STRING = "package io.pivotal; "
        + "public class TestPartitionResolver implements org.apache.geode.cache.PartitionResolver { "
        + "   @Override" + "   public void close() {" + "   }" + "   @Override"
        + "   public Object getRoutingObject(org.apache.geode.cache.EntryOperation opDetails) { "
        + "    return null; " + "   }" + "   @Override" + "   public String getName() { "
        + "    return \"TestPartitionResolver\";" + "   }" + " }";

    final File prJarFile = new File (tmpDir.getRoot(), "myPartitionResolver.jar");
    new JarBuilder().buildJar(prJarFile, PR_STRING);
```